train.py

```diff
@@ -94,34 +119,39 @@
 
             # Training
             num_batches = X_train.shape[0] // BATCH_SIZE
-            for b in range(num_batches):
+            for b in tqdm(range(num_batches)):
                 x_batch, y_batch = next(train_batch_generator)
                 seq_len = np.array([list(x).index(0) + 1 for x in x_batch])  # actual lengths of sequences
-                loss_tr, acc, _ = sess.run([loss, accuracy, optimizer],
-                                           feed_dict={batch_ph: x_batch,
-                                                      target_ph: y_batch,
-                                                      seq_len_ph: seq_len,
-                                                      keep_prob_ph: KEEP_PROB})
+                loss_tr, acc, _ , summary= sess.run([loss, accuracy, optimizer, merged],
+                                                    feed_dict={batch_ph: x_batch,
+                                                    target_ph: y_batch,
+                                                    seq_len_ph: seq_len,
+                                                    keep_prob_ph: KEEP_PROB})
                 accuracy_train += acc
                 loss_train = loss_tr * DELTA + loss_train * (1 - DELTA)
+                train_writer.add_summary(summary, b)
```

Comment:
Probably it should be `b + num_batches * epoch` instead of `b` here for correct continuous curves on tensorboard